build.zig.zon

```diff
@@ -1,6 +1,9 @@
 .{
     .name = "zshuffle",
     .version = "0.0.0",
+    .paths = .{
+        ".",
+    },
```

Comment:
This is a backwards-compatible change with Zig `0.11.0` but allows us to build with the latest Zig `master` (`0.12.0-dev.1676+40b8c993f`)